### ISSUE TITLE
Create canaries and alerting for digital-land key tables

### ISSUE THREAD

**Issue by Ben-Hodgkiss** [digital-land/technical-documentation#91]
**Overview**

Following an incident where some of the digital-land databases in datasette were unavailable following a change to digital_land.sqlite3, we have agreed to introduce canaries and alerts to key tables in the digital-land database.

**Tech Approach**

- ‌

**Acceptance Criteria/Tests**

- The tables named above should have regular canaries checking availability.
- If the table is unavailable, an alert should be produced so that the relevant team is made aware of the issue.

**Comment by cpcundill** [comment]
@Ben-Hodgkiss and @eveleighoj My assumption here is that we want to start managing the canary code for datasette within the GitHub repository (as per digital-land.info).  The canary code will need to be configured to hit additional datasette URLs which exercise known tables, e.g. should it need to check if the issues tables is present, it would make a GET request to https://datasette.planning.data.gov.uk/digital-land/issue

Am I on the right track?  If so, please could you provide a list of databases and tables therein for to be included for checking?

**Comment by eveleighoj** [comment]
Yep that sounds correct. A simple get request to the necessary urls will be able to check that. the datasette service is up and running. e.g:

* check organisation page is working - https://datasette.planning.data.gov.uk/digital-land/organisation
* check issues page is working - https://datasette.planning.data.gov.uk/digital-land/issue 

We may not need an expansive list for now but we can add them as we go.
